types/three/three-core.d.ts

```diff
@@ -6644,7 +6644,7 @@ export class IcosahedronGeometry extends PolyhedronGeometry {
 }
 
 export class LatheBufferGeometry extends BufferGeometry {
-    constructor(points: Vector3[], segments?: number, phiStart?: number, phiLength?: number);
+    constructor(points: Vector2[], segments?: number, phiStart?: number, phiLength?: number);
 
     parameters: {
         points: Vector3[];
```

Comment:
Should `points` here  and at L6661 also be `Vector2[]`?

Reply:
Yep - `LatheGeometry` docs also specify Vector2.  Under the hood, `LatheGeometry` uses `LatheBufferGeometry`.

https://threejs.org/docs/#api/geometries/LatheGeometry
https://threejs.org/docs/#api/geometries/LatheBufferGeometry

Reply:
Updated source doc link

Reply:
Cool, thanks!

Do you mind making the change at L6650 and L6651 from `points: Vector3[];` to `points: Vector2[];`?

Reply:
nice catch :) updated